### DISCUSSION TITLE
CVE-2024-21626 and OKD

### DISCUSSION THREAD

**Discussion by nate-duke** [okd-project/okd#1877] [General]
https://access.redhat.com/security/cve/cve-2024-21626

We're starting to get notices from our security folks about this one. Is there any hope that we'll see a fix/backport of stuff that will surely get done in OCP down here in OKD?

If so, which versions could possibly see an update?

**Comment by vrutkovs** [comment]
We can only baclkport to latest stable stream - 4.15 - once https://bugzilla.redhat.com/show_bug.cgi?id=2262166 fix lands in FCOS stable.

However, it can be manually fixed in 4.13+ by making a custom OS - https://docs.okd.io/4.14/post_installation_configuration/coreos-layering.html, including RPM (or a binary) with the fix
